src/App.vue

```diff
@@ -566,7 +570,8 @@ export default {
       showBlipGroupCard: true,
       disableLink: false,
       editable: true,
-      deletable: true
+      deletable: true,
+      readonly: this.readonly
```

Comment:
Shouldn't here be the default value, just like the other properties receive?

Reply:
I've setted false as default value